collector/bgp.go

```diff
@@ -340,6 +359,41 @@ func getPeerAdvertisedPrefixes(ch chan<- prometheus.Metric, wg *sync.WaitGroup,
 	newGauge(ch, bgpDesc["prefixAdvertisedCount"], float64(advertisedPrefixes.TotalPrefixCounter), peerLabels...)
 }
 
+type bgpRoutes struct {
+	// We care only about the routes
+	Routes map[string][]json.RawMessage `json:"routes"`
+}
+
+func getPeerAcceptedRoutes(AFI string, SAFI string, vrfName string, neighbor string) (float64, error) {
+	var cmd string
+	if strings.ToLower(vrfName) == "default" {
+		cmd = fmt.Sprintf("show bgp  %s %s neighbors %s routes json", strings.ToLower(AFI), strings.ToLower(SAFI), neighbor)
+	} else {
+		cmd = fmt.Sprintf("show bgp vrf %s %s %s neighbors %s routes json", vrfName, strings.ToLower(AFI), strings.ToLower(SAFI), neighbor)
+	}
+
+	output, err := executeBGPCommand(cmd)
+	if err != nil {
+		return 0, err
+	}
+
+	var routes bgpRoutes
```

Comment:
This is unused and a duplicate of `processPeerAcceptedRoutes`. I think it's better to keep it within the function, to keep it consistent with `getPeerAdvertisedPrefixes`.